src/pages/Home.js

```diff
@@ -1,8 +1,19 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import Products from '../components/Products'
+import { useLoaderData } from 'react-router-dom'
 
 const Home = () => {
+    const [products, setProducts] = useState([]);
```

Comment:
Formatting: const [ products, setProducts ].
Let's add a ticket to add an auto formatting on save.

Reply:
my auto formatting forces [products, setProducts] lol